epoch/api/README.md

```diff
@@ -26,6 +27,10 @@ The epoch node exposes the following APIs:
     * It is defined in the rest of this document;
     * It is **not** meant to be exposed on the Internet;
     * Its TCP port is configurable.
+  * External Channels WebSocket endpoint.
```

Comment:
Could you please rephrase the sentence above "User API. It consists of three TCP endpoints:" to "User API. It consists of the following TCP endpoints:" or analogous?

Reply:
I see this addressed.